langchain4j-core/src/main/java/dev/langchain4j/data/message/RelevantAiMessage.java

```diff
@@ -0,0 +1,67 @@
+package dev.langchain4j.data.message;
+
+import java.util.Objects;
+
+import static dev.langchain4j.data.message.ChatMessageType.Relevant;
+import static dev.langchain4j.internal.Utils.quoted;
+
+public class RelevantAiMessage extends ChatMessage {
```

Comment:
What is the difference between `RelevantMessage` and `RelevantAiMessage`?

Reply:
RelevantAiMessage is useless  i will remove